<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMutualismBreakdownerParametersWidget</class>
 <widget class="QWidget" name="QtMutualismBreakdownerParametersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="2" column="0" colspan="2">
    <widget class="QToolBox" name="toolBox">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_spatial_plot">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>174</width>
        <height>105</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Spatial plot</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QLabel" name="label_spatial_height">
         <property name="text">
          <string>Grid height</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="box_spatial_width">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_spatial_width">
         <property name="text">
          <string>Grid width</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="box_spatial_height">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="image_spatial">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="korter_project.qrc">:/images/Grid.jpg</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_initial_conditions">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>386</width>
        <height>357</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Initial conditions</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="2" column="0">
        <widget class="QLabel" name="label_n_seeds">
         <property name="text">
          <string>Number of seeds</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_n_nurse_plants">
         <property name="text">
          <string>Number of nurse plants</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="image_initial_conditions">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="korter_project.qrc">:/images/PrimordialSoup.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="box_n_nurse_plants"/>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="box_n_seeds"/>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="box_rng_seed"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_initial_seagrass_concentration">
         <property name="text">
          <string>RNG seed</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_fitness">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>82</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Fitness</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2"/>
     </widget>
     <widget class="QWidget" name="page_seagrass_colonisation">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Seagrass colonisation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11"/>
     </widget>
     <widget class="QWidget" name="page_seagrass_growth">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Seagrass growth</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5"/>
     </widget>
     <widget class="QWidget" name="page_sulfide_consumption">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Sulfide consumption</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7"/>
     </widget>
     <widget class="QWidget" name="page_sulfide_detoxification">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Sulfide detoxification</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6"/>
     </widget>
     <widget class="QWidget" name="page_sulfide_diffusion">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Sulfide diffusion</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
     <widget class="QWidget" name="page_sulfide_production">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Sulfide production</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3"/>
     </widget>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="button_save">
     <property name="text">
      <string>&amp;Save parameters</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="button_load">
     <property name="text">
      <string>&amp;Load parameters</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>button_load</tabstop>
  <tabstop>button_save</tabstop>
 </tabstops>
 <resources>
  <include location="korter_project.qrc"/>
 </resources>
 <connections/>
</ui>
